lib/checks/navigation/region-after.js

```diff
@@ -0,0 +1,24 @@
+function regionAfter(results) {
```

Comment:
I think we need to create some unit tests for this after method. That'll be much easier than trying to pack all the logic of this into integration tests.